src/queries/queries.js

```diff
@@ -504,3 +504,14 @@ export const buildAccountQueryByLogin = login => ({
     as: `${ACCOUNTS_DOCTYPE}/account_type/login/${login}`
   }
 })
+
+export const buildFirstTimeserieQuery = () => {
```

Comment:
I don't think it's necessary to define a new query for this, I think the exisiting `buildAggregatedTimeseriesQuery` should do the job, by passing a limit=1 ?  https://github.com/cozy/coachCO2/blob/90394eb67921cb1c8fd229299aedee7572df4879/src/queries/queries.js#L28
And if you really need the _id, you can add it in the existing query (but I don't see the _id being used)

Reply:
Actually you never need to specify the _id in the .select, but it's included by default: https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-client/src/DocumentCollection.js#L542

Reply:
Interesting. Works very well 👍